easybuild/easyconfigs/g/GMT/GMT-5.1.2_netCDF.patch

```diff
@@ -0,0 +1,23 @@
+#Do not check netCDF LD_LIBRARY_PATH dependencies using nc-config --libs, as we provide with EB
```

Comment:
`nc-config --libs` gives correct info, but does not give all the path.
```
nc-config --libs
-L/apps/brussel/ivybridge/software/netCDF/4.3.3.1-intel-2015a/lib -lnetcdf -lhdf5_hl -lhdf5 -lsz -lz -ldl -lm -liomp5 -lpthread -lz -lcurl
```
So cmake does not find `libcurl` and `libiomp5` (and maybe others, but I do not remember). netCDF does not check the LD_LIBRARY_PATH, so instead of programming to check n the `.cmake` file, I did choose the easy way, because provide all the necessary libraries via LD_LIBRARY_PATH. And anyway, cmake use those extra libs, and we provide the LD_LIBRARY_PATH, so no linkong problems.